src/main/java/bio/terra/workspace/service/iam/model/IamRole.java

```diff
@@ -0,0 +1,42 @@
+package bio.terra.workspace.service.iam.model;
+
+import com.google.common.collect.BiMap;
+import com.google.common.collect.EnumBiMap;
+import com.google.common.collect.EnumHashBiMap;
+
+/** Internal representation of IAM roles. */
+public enum IamRole {
+  READER,
+  WRITER,
+  OWNER;
+
+  private static final BiMap<IamRole, String> samMap = EnumHashBiMap.create(IamRole.class);
+  private static final BiMap<IamRole, bio.terra.workspace.generated.model.IamRole> apiMap =
```

Comment:
Technically this isn't used in this change and could go in #164 instead - I just left it here since I'm expected both PRs to go in at similar times.